bugbug/_classification.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import numpy as np
+from imblearn.metrics import (
+    geometric_mean_score,
+    make_index_balanced_accuracy,
+    specificity_score,
+)
+from sklearn.metrics.classification import precision_recall_fscore_support
+from sklearn.utils.multiclass import unique_labels
+
+
+def classification_report_imbalanced_values(
+    y_true,
+    y_pred,
+    labels=None,
+    target_names=None,
+    sample_weight=None,
+    digits=2,
+    alpha=0.1,
+):
+    """Reimplementation of imblearn.metrics.classification_report_imbalanced
```

Comment:
Whoah, I thought we could access the raw data somehow :(

How bad would it be to parse the output table?

The other option is to just store precision and recall, so we don't have to reimplement all. In the end, that's what I mostly use.

Reply:
I've removed some extra code in `classification_report_imbalanced_values` that were not needed. I think the amount of code duplication is acceptable now.

I'm more than hesitant to parse the output table, we will likely hit some corner-cases which will makes the parsing code non trivial, we might parse the output wrongly and any dependency update can breaks the parsing either loudly or worse silently.